JS/Lesson16/sinan/script.js

```diff
@@ -1,142 +1,177 @@
 // 1. Write a loop that prints out the numbers from 1 to 10.
-for (let i = 1; i<=10; i++) {
+for (let i =0; i<10; i++) {
```

Comment:
```suggestion
for (let i =1; i<11; i++) {
```
Otherwise numbers from 0 to 9 are printed!